README.md

```diff
@@ -1,4 +1,4 @@
-#Portfolio-mobile-version 
+#Portfolio-mobile-version
```

Comment:
- [ ] [OPTIONAL] Kindly check the alignment in your Authors section;
![image](https://user-images.githubusercontent.com/61408860/171449632-be6f5465-98da-46e5-8e1a-a55614ec1d9a.png)